plugins/obs-ffmpeg/obs-ffmpeg-vaapi.c

```diff
@@ -511,18 +512,73 @@ static void set_visible(obs_properties_t *ppts, const char *name, bool visible)
 
 static void vaapi_defaults(obs_data_t *settings)
 {
-	obs_data_set_default_string(settings, "vaapi_device",
-				    "/dev/dri/renderD128");
+	const char *device = vaapi_get_h264_default_device();
+	VADisplay va_dpy;
```

Comment:
Declare `va_dpy` where it is assigned instead.

Reply:
Changes done waiting for your re-review before squashing.